audiofile/core/info.py

```diff
@@ -1,6 +1,7 @@
 """Read, write, and get information about audio files."""
 import logging
```

Comment:
```suggestion
```

Reply:
we can also remove `import audeer` a few lines later

Reply:
and there are a few other unused imports:

* info.py -> logging
* io.py -> sys, warnings
* utils.py -> os

Reply:
Removed those imports.

`import audeer` is needed in `info.py`

Reply:
Right, I had actually opened `convert.py`. There is an unused `import audeer` and another `import logging`. And two more things I noticed when checking the imports:

* utils.py: `shlex` should be imported before `subprocess` for alphabetical ordering
* io.py: starts with `from __future__ import division` and I don't see any usage of `division`. Or does this has some special  meaning?

Reply:
I removed all the suggested imports. `from __future__ import division` was from the early beginning when I still targeted Python 2.7 support.